BottomBar.Droid/Renderers/BottomBarPageRenderer.cs

```diff
@@ -227,7 +231,8 @@ void UpdateBarTextColor ()
 				return;
 			}
 
-			// haven't found yet how to set text color for tab items on_bottomBar, doesn't seem to have a direct way
+            _bottomBar.SetActiveTabColor(Element.BarTextColor.ToAndroid());

```

Comment:
and the others on the project?And the other ones on the project? Normally it should not be a problem between XS and VS I work on another OSS project where it's not a problem